modules/ui/TopBar.py

```diff
@@ -103,25 +107,11 @@ def __create_training_method(self):
                 ("Embedding", TrainingMethod.EMBEDDING),
                 ("Fine Tune VAE", TrainingMethod.FINE_TUNE_VAE),
             ]
-        elif self.train_config.model_type.is_stable_diffusion_3():
-            values = [
-                ("Fine Tune", TrainingMethod.FINE_TUNE),
-                ("LoRA", TrainingMethod.LORA),
-                ("Embedding", TrainingMethod.EMBEDDING),
-            ]
-        elif self.train_config.model_type.is_stable_diffusion_xl():
-            values = [
-                ("Fine Tune", TrainingMethod.FINE_TUNE),
-                ("LoRA", TrainingMethod.LORA),
-                ("Embedding", TrainingMethod.EMBEDDING),
-            ]
-        elif self.train_config.model_type.is_wuerstchen():
-            values = [
-                ("Fine Tune", TrainingMethod.FINE_TUNE),
-                ("LoRA", TrainingMethod.LORA),
-                ("Embedding", TrainingMethod.EMBEDDING),
-            ]
-        elif self.train_config.model_type.is_pixart():
+        elif (
+            self.train_config.model_type.is_stable_diffusion_3()
```

Comment:
In this case I would prefer to have all 4 conditions each on a new line. But the last line has two conditions. Is it possible to set the formatter up so it doesn't join manually broken lines? That way we could manually insert a line break